<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>BinnedMapDialogWidget</class>
  <widget class="QWidget" name="BinnedMapDialogWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>320</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form1</string>
    </property>
    <layout class="QGridLayout">
      <item row="0" column="0">
        <widget class="Q3GroupBox" name="groupBox1">
          <property name="title">
            <string>Settings</string>
          </property>
          <layout class="QGridLayout">
            <item row="1" column="0">
              <widget class="QLabel" name="textLabel1_2">
                <property name="text">
                  <string>Y:</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="textLabel1_3">
                <property name="text">
                  <string>Z:</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1" rowspan="1" colspan="7">
              <widget class="VectorSelector" name="_Z"/>
            </item>
            <item row="1" column="1" rowspan="1" colspan="7">
              <widget class="VectorSelector" name="_Y"/>
            </item>
            <item row="0" column="1" rowspan="1" colspan="7">
              <widget class="VectorSelector" name="_X"/>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="textLabel1">
                <property name="text">
                  <string>X:</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="6" column="0" rowspan="1" colspan="8">
              <widget class="Line" name="line1">
                <property name="frameShape">
                  <enum>QFrame::HLine</enum>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                </property>
              </widget>
            </item>
            <item row="8" column="0" rowspan="1" colspan="2">
              <widget class="QLabel" name="textLabel4_2">
                <property name="text">
                  <string>Hits Map:</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0" rowspan="1" colspan="3">
              <widget class="QLabel" name="textLabel3">
                <property name="text">
                  <string>X Binning: From</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="3">
              <widget class="QLineEdit" name="_Xmin">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>-1</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item row="3" column="4">
              <widget class="QLabel" name="TextLabel4_5_2_2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>To:</string>
                </property>
                <property name="buddy" stdset="0">
                  <cstring>Max</cstring>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="5">
              <widget class="QLineEdit" name="_Xmax">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>1</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item row="3" column="6">
              <widget class="QLabel" name="TextLabel5_2_2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>Num X bins:</string>
                </property>
                <property name="buddy" stdset="0">
                  <cstring>N</cstring>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="7">
              <widget class="QSpinBox" name="_Xn">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="maximum">
                  <number>1000000</number>
                </property>
                <property name="singleStep">
                  <number>5</number>
                </property>
                <property name="value">
                  <number>60</number>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The number of bins in the Histogram.</string>
                </property>
              </widget>
            </item>
            <item row="4" column="0" rowspan="1" colspan="3">
              <widget class="QLabel" name="textLabel3_2">
                <property name="text">
                  <string>Y Binning: From</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="3">
              <widget class="QLineEdit" name="_Ymin">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>-1</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item row="4" column="4">
              <widget class="QLabel" name="TextLabel4_5_2_2_2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>To:</string>
                </property>
                <property name="buddy" stdset="0">
                  <cstring>Max</cstring>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="5">
              <widget class="QLineEdit" name="_Ymax">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>1</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item row="4" column="6">
              <widget class="QLabel" name="TextLabel5_2_2_2">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>Num Y bins:</string>
                </property>
                <property name="buddy" stdset="0">
                  <cstring>N</cstring>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="7">
              <widget class="QSpinBox" name="_Yn">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="maximum">
                  <number>1000000</number>
                </property>
                <property name="singleStep">
                  <number>5</number>
                </property>
                <property name="value">
                  <number>60</number>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>The number of bins in the Histogram.</string>
                </property>
              </widget>
            </item>
            <item row="5" column="0" rowspan="1" colspan="2">
              <widget class="QPushButton" name="_AutoBin">
                <property name="sizePolicy">
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text">
                  <string>A&amp;uto Bin</string>
                </property>
                <property name="shortcut">
                  <string>Alt+U</string>
                </property>
                <property name="whatsThis" stdset="0">
                  <string>Set the upper and lower limits, and the number of bins to a reasonable default.</string>
                </property>
              </widget>
            </item>
            <item row="5" column="2" rowspan="1" colspan="4">
              <widget class="QCheckBox" name="_realTimeAutoBin">
                <property name="text">
                  <string>&amp;Real-time auto bin</string>
                </property>
                <property name="shortcut">
                  <string>Alt+R</string>
                </property>
              </widget>
            </item>
            <item row="7" column="2" rowspan="1" colspan="6">
              <widget class="QLineEdit" name="_binnedMap"/>
            </item>
            <item row="8" column="2" rowspan="1" colspan="6">
              <widget class="QLineEdit" name="_hitsMap"/>
            </item>
            <item row="7" column="0" rowspan="1" colspan="2">
              <widget class="QLabel" name="textLabel4">
                <property name="text">
                  <string>Binned Map:</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignVCenter|Qt::AlignRight</set>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local">kstcombobox.h</include>
    <include location="local">kstcombobox.h</include>
    <include location="local">kstcombobox.h</include>
  </includes>
</ui>
